static/katex.less

```diff
@@ -232,8 +232,25 @@
         & + .mop.mtight { margin-left: @thinspace; }
     }
 
+    .vlist-t {
+        display: inline-table;
+        table-layout: fixed;
```

Comment:
Motivated by the fact that the other table displays in katex.less have table-layout: fixed.